app/models/backend_api_slug.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class BackendApiSlug < ApplicationRecord
```

Comment:
Hmm. We may want to later grow this to more than just a "slug", incorporating other attributes that belong to the association between `Service` and `BackendApi`. Also, we agreed that one backend API (and no more than one) used in a service is allowed to have an empty slug. So I think we should rename this model to something else.

Reply:
Right for the naming I do not know what to put.
Slug would never be empty but will be `"/"`

Reply:
I think `slug` should be `path` instead 
And we can use `BackendApiConfig` as name, what do you think @guicassolato ?

Reply:
`BackendApiConfig` 👍